eyewitness: add page (#9298)
tldr-pages/tldr: pages/linux/eyewitness.md

> # eyewitness

Reviewer: ```suggestion
# EyeWitness
```

> > Python based tool to gather screenshots of websites, as well as additional information such as header info and default credentials if possible.

Reviewer 2: ```suggestion
> Python-based tool to gather screenshots of websites, as well as additional information such as header info and default credentials if possible.
```

> > More information: <https://www.christophertruncer.com/eyewitness-triage-tool>.

Reviewer 2: ```suggestion
> More information: <https://github.com/FortyNorthSecurity/EyeWitness>.
```
I think it is better to link to their GitHub page as their usage guides seems to be updated frequently.

> - Scan a single website using --single flag:

Reviewer 2: ```suggestion
- Scan a single website using `--single` flag:
```

> - Scan a single website using `--single` flag:

Reviewer 2: ```suggestion
- Scan a single website using the `--single` flag:
```

> - Scan a single website using `--single` flag:

Reviewer 3: ```suggestion
- Scan a specific website:
```

- Optional fix: `a specific` may sound better.
- Optional fix: there is no need to explicitly mention flag names until it's unclear (definition is missing in our guides).

> `eyewitness --single {{website}}`

Reviewer: The command has this case
```suggestion
`EyeWitness --single {{website}}`
```

Reviewer 2: > The command has this case

Yes it is mentioned here https://www.christophertruncer.com/eyewitness-2-0-release-and-user-guide/

Also, `.py` file extension is used

Author: Using the version from the Kali repos, it does not require specific capitalization nor the python extension. If we amend the TLDR to reflect this capitalization it will not run on the commandline.

Reviewer: in that case we should add an alias page.

> `EyeWitness --single {{website}}`

Reviewer 2: ```suggestion
`EyeWitness --single {{website_url}}`
```
Would this be better?

> `EyeWitness --single {{website}}`

Reviewer 3: ```suggestion
`EyeWitness --single "{{website_url}}"`
```

Option fix: possible missing quotes: quotes can be added if the placeholder can contain some special characters for POSIX-like shells.

> - Scan multiple urls from file and output to foldername:

Reviewer 2: ```suggestion
- Scan multiple URLs from a file and output to a specific directory:
```

> `eyewitness -f {{filename}} -d {{foldername}}`

Reviewer 2: ```suggestion
`eyewitness -f {{path/to/file}} -d {{path/to/directory}}`
```

Reviewer 3: Note for the PR author: it's a path placeholder format [violation](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): mandatory fix.

> - Scan muliple urls from xml file (nmap and nessus xml file format) and output to folername:

Reviewer 2: ```suggestion
- Scan multiple URLs from XML file (`nmap` and `nessus` XML file format) and output to a specific directory:
```

> `eyewitness -x {{filename}} -d {{foldername}}`

Reviewer 2: ```suggestion
`eyewitness -x {{path/to/file.xml}} -d {{path/to/directory}}`
```

> - Scan multiple urls from file providing a specific useragent such as Mozilla/4.0:

Reviewer 2: ```suggestion
- Scan multiple URLs from a file providing a specific user agent such as Mozilla/4.0:
```

> `eyewitness -f {{filename}} --useragent {{Mozilla/4.0}}`

Reviewer 2: ```suggestion
`eyewitness -f {{path/to/file}} --useragent {{Mozilla/4.0}}`
```

> - Scan multiple urls from file using --jitter flag to randomize accessing each url with a base number of seconds:

Reviewer 2: ```suggestion
- Scan multiple URLs from file using the `--jitter` flag to randomize accessing each URL with a base number of seconds:
```

> - Scan multiple URLs from file using the `--jitter` flag to randomize accessing each URL with a base number of seconds:

Reviewer 2: ```suggestion
- Scan multiple URLs from a file using the `--jitter` flag to randomize accessing each URL with a base number of seconds:
```

> `eyewitness -f {{filename}} --jitter {{seconds}}`

Reviewer 2: ```suggestion
`eyewitness -f {{path/to/file}} --jitter {{seconds}}`
```
